main.go

```diff
@@ -1,27 +1,74 @@
 package main
 
 import (
-  "github.com/gin-gonic/gin"
-  "github.com/distributed-marketplace-system/controllers"
-  "github.com/distributed-marketplace-system/db"
-  "github.com/joho/godotenv"
-  "github.com/distributed-marketplace-system/util"
-  _ "fmt"
+	"log"
+	"net/http"
+	"os"
+	"runtime"
+
+	"github.com/distributed-marketplace-system/controllers"
+	"github.com/distributed-marketplace-system/db"
+
+	"github.com/distributed-marketplace-system/util"
+	"github.com/gin-gonic/gin"
+	"github.com/joho/godotenv"
 )
 
-func main() {
-  err := godotenv.Load(".env")
+func routes() {
+	router := gin.Default()
+
+	// Load the static files
+	router.LoadHTMLGlob("./public/html/*")
+	router.Static("/public", "./public")
+
+	// User APIs
+	user_r := router.Group("/user")
+	{
+		user := new(controllers.UserController)
 
-  if err != nil {
-    panic("Fatal Error: Couldn't loading .env file")
-  }
+		user_r.GET("/get_all", user.GetAll)
+		user_r.GET("/get_one/:id", user.GetOne)
+		user_r.POST("/register", user.RegisterUser)
+		user_r.POST("/login", user.LoginUser)
```

Comment:
I'm not sure how these routes will work
if they will work like this 
```
GET  /user/get_all
GET /user/get_one/:id
```
Then they are not quite right
We should make the best use of HTTP verbs and status
for example
```
GET /users          # return all users
GET /user/:id       # return single user
GET /products       # return all products
GET /products/:id   # return single product
POST /products      # create new product 
PUT /products       # update product 
DELETE /products    # delete product 
```
As login and register are special endpoint 
we can separate them
```
POST /login
POST /register
```
For reference, you can check [this](https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/)

Reply:
GET /user**s**/:id       # return single user

Always use plural nouns.